components/sidebar/sidebar_menu/sidebar_menu.scss

```diff
@@ -0,0 +1,8 @@
+// Manual override of the global menu (and item) styling for the sidebar dropdown menu
+#sidebarDropdownMenu .MenuItem {
+    > button,
+    > div,
+    > a {
```

Comment:
I would go against styling elements directly, since this is the reason why our styles are conflicting all over webapp, since the scope they are in has a lower specificity than the element styles. I know it can be annoying to add classes to every element but until we sort out the structure of our CSS, maybe use help from styled components we will run into issues with other elements in the future. We don't know where in the app a button, div or link might get updated with this :D

Reply:
I agree, but the HTML structure currently does not provide classes to the elements. 

Should we adjust the component in this case, so that they gain classes to better target them?

BTW: there should never be a override with this kind of selector since the id takes precedence over any class-based selector, but fair enough, you never know what happens in the future. 🕦 😃

Reply:
Oh I didn't notice its an ID, then I think we should be good!

Reply:
I just double-checked and the way the menu is rendered is pretty over-complicated, so adding a class to each of the items would mean to change at least 6-8 files (if not more). So I think we should be good for now with the id-selector! :D